README.md

```diff
@@ -2,7 +2,7 @@
 [![Gitter chat](https://badges.gitter.im/polynote/polynote.png)](https://gitter.im/polynote/polynote)
 [![Build status](https://github.com/polynote/polynote/workflows/Build/badge.svg)](https://github.com/polynote/polynote/workflows/Build)
 
-Polynote is an experimental polyglot notebook environment. Currently, it supports Scala and Python (with or without Spark),
+Polynote is an experimental polyglot notebook environment. Currently, it supports Scala, Python (with or without Spark),
```

Comment:
I think this changes the semantics of the sentence – "with or without Spark" refers to both Scala and Python, which is why they were grouped together instead of both being part of the comma list.